media-libs/libplacebo/libplacebo-0.5.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Why two versions?

Reply:
No reason, other than "I had them in my own tree at the time of submitting". I'll remove the version 0.5.